images/nginx/rootfs/Dockerfile

```diff
@@ -64,6 +64,7 @@ RUN apk update \
   /var/lib/nginx/scgi \
   /var/lib/nginx/uwsgi \
   /var/log/audit \
+  /tmp/nginx \
```

Comment:
Why we need this dir?

Reply:
same as in ingress-controller image, I wanted to use it for pid file /tmp/nginx/nginx.pid